Update/some_004.txt

```diff
@@ -2656,7 +2656,7 @@ void main()
 //r自分でも制御不可能なほど支離滅裂な早口をまくし立てて、あまつさえ私は弁明というよりも、後ろめたい気持ちを打ち消したくてただ、夢中で言葉を羅列してゆく。だけど、
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "自分でも制御不可能なほど支離滅裂な早口をまくし立てて、あまつさえ私は弁明というよりも、後ろめたい気持ちを打ち消したくてただ、夢中で言葉を羅列してゆく。だけど、",
-			NULL, "I muffled my incoherently rambling mouth. Not to be defensive, but because I wanted to put away those feelings. However,", Line_Normal);
+			NULL, "I rambled on incoherently, unable to control the flow of my words. I wasn't trying to be defensive, I just wanted to get over my guilty conscience so I kept sputtering. However,", Line_Normal);
```

Comment:
```suggestion
			NULL, "I rambled on incoherently, unable to control the flow of my words. I wasn't trying to be defensive. I just wanted to get over my guilty conscience so I kept sputtering. However,", Line_Normal);
```
Tiny comma splice. Would normally advise a semicolon, but it's not part of Higurashi style — so a period instead. :)